src/v2/Apps/Show/Components/ShowViewingRoom.tsx

```diff
@@ -1,22 +1,82 @@
 import React from "react"
-import { Box, BoxProps, ResponsiveBox, Text } from "@artsy/palette"
+import { Box, BoxProps, MediumCard, ResponsiveBox, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowViewingRoom_show } from "v2/__generated__/ShowViewingRoom_show.graphql"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { crop } from "v2/Utils/resizer"
+import { getTagProps } from "v2/Components/ViewingRoomCard"
 
-interface ShowViewingRoom extends BoxProps {}
+interface ShowViewingRoomProps extends BoxProps {
+  show: ShowViewingRoom_show
+}
+
+export const ShowViewingRoom: React.FC<ShowViewingRoomProps> = ({
+  show,
+  ...rest
+}) => {
+  const [{ node: viewingRoom }] = show.viewingRoomsConnection.edges
+
+  const thumbnail = crop(viewingRoom.image.imageURLs.normalized, {
+    width: 900,
+    height: 1200,
```

Comment:
This bums me out:
* No 'resized/crop' support in Gravity (which would be easy enough to add)
* No real image support in the card component (I'll take a crack at this, we shouldn't really have components in there that have opinions on how images are loaded, IMO)

Reply:
> No 'resized/crop' support in Gravity

In gravity? I don't totally follow.

Reply:
As in it'd make sense to either support an image interface in Gravity's GraphQL endpoint that's similar to Metaphysics' — or to consolidate all the resizing logic into a function Force (add srcSet, deprecate Metaphysics')